Cargo.toml

```diff
@@ -9,7 +9,7 @@ members = [
 ]
 
 [workspace.dependencies]
-tantivy = "0.22.0"
+tantivy = "0.21.0"
```

Comment:
Why did we have to revert the update?

Reply:
That's an outdated commit, I did the update on main and merged it back into this to separate it, check the latest one :)